lib/lambda_ethereum_consensus/execution/execution_chain.ex

```diff
@@ -266,4 +269,15 @@ defmodule LambdaEthereumConsensus.Execution.ExecutionChain do
 
   defp slots_per_eth1_voting_period(),
     do: ChainSpec.get("EPOCHS_PER_ETH1_VOTING_PERIOD") * ChainSpec.get("SLOTS_PER_EPOCH")
+
+  defp persist_execution_state(state) do
+    put("", state)
+  end
+
+  defp fetch_execution_state(), do: get("")
+
+  defp fetch_execution_state!() do
```

Comment:
Can we add a spec to the get and persist functions so it gives a clear indication of what shape the "execution_state" has?

Reply:
Good idea!